src/api/users/getUsers.js

```diff
@@ -1,6 +1,14 @@
 /* @flow */
-import type { Auth, User } from '../../types';
+import type { Auth, ApiResponseSuccess, User } from '../apiTypes';
 import { apiGet } from '../apiFetch';
 
-export default (auth: Auth): Promise<User[]> =>
-  apiGet(auth, 'users', res => res.members, { client_gravatar: true });
+/** See https://zulipchat.com/api/get-all-users */
+
+type ApiResponseUsers = ApiResponseSuccess & {
+  members: User[],
+};
+
+export default (auth: Auth, clientGravatar: boolean = true): Promise<ApiResponseUsers> =>
+  apiGet(auth, 'users', (res: ApiResponseUsers) => res, {
```

Comment:
This changes the API this function provides to callers, right -- won't that break something?

... Aha, it doesn't -- because this function has no callers.

Good then, but that's important information for the reader in the commit message. :-)

Reply:
I did update the message.